pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.aquality-automation</groupId>
     <artifactId>aquality-selenium</artifactId>
-    <version>2.0.2</version>
+    <version>2.0.3</version>
```

Comment:
Increase minor version because it is a new feature. So, version should be 2.1.0

Reply:
fixed